webpki-root-certs/LICENSE-MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2023 Dirkjan Ochtman <dirkjan@ochtman.nl>
```

Comment:
I think in other repos we use symlinks to the parent dir's `LICENSE` files.

Reply:
Replaced with symlinks in a separate commit since this change needed to be done for all three crates and the workspace.

Reply:
Oops, messed up the links. Will fix.

Reply:
Ok I think I've got it working now. GitHub doesn't follow the links in the web view but a clean checkout did the right thing.